markdown/org/docs/patterns/lunetius/fabric/en.md

```diff
@@ -1,6 +1,15 @@
-Lunetius can be made out of almost any woven fabric. Historically "accurate" would be linen, wool, and, to an extent, cotton. If you care about authenticity, research this a bit. In any case, natural fibers without any stretch are the way to go.
+---
+title: Fabric
```

Comment:
Please use `Fabric options` as title because that's what we use everywhere.

Reply:
Done too!